exes/mqtt-bridge/inc/structs.hpp

```diff
@@ -1,18 +1,15 @@
 #pragma once
 
 #include <any>
+#include <cstdint>
 #include <optional>
+#include <string>
 
-#include <tfc/ipc.hpp>
+#include <sparkplug_b/sparkplug_b.pb.h>
 
-namespace tfc::mqtt::structs {
-
-enum struct ssl_active_e { yes, no };
-
-struct signal_data {
-  tfc::ipc_ruler::signal information;
-  tfc::ipc::details::any_slot receiver;
-  std::optional<std::any> current_value;
+struct spark_plug_b_variable {
+  std::string name;
+  org::eclipse::tahu::protobuf::DataType datatype;
```

Comment:
is it needed to store the datatype here? can it not be deduced from the value itself?

Reply:
if the optional is empty then it is impossible to know what the datatype is. I also don´t see a problem with having the struct maintain all needed information about each variable